pkg/lib/bundle/validate.go

```diff
@@ -45,8 +45,10 @@ type imageValidator struct {
 // PullBundleImage shells out to a container tool and pulls a given image tag
 // Then it unpacks the image layer filesystem contents and pushes the contents
 // to a specified directory for further validation
-func (i imageValidator) PullBundleImage(imageTag, directory string) error {
+func (i imageValidator) PullBundleImage(containerTool, imageTag, directory string) error {
```

Comment:
If the first argument were an `ImageReader`, it would be easier to test. The fact that this implementation instantiates its own ImageReader instead of having one injected was causing the trouble with the now-deleted tests that Nick pointed out earlier.